train of pairs: Use block-wise terminology (#11)
anr-bmbf-pivot/draft-dns-over-coap: draft-lenders-dns-over-coap.md

DNS Responses in CoAP Responses
-------------------------------

Each DNS query-response pair is mapped to a CoAP REST request-response
operation, which may consist of several CoAP request-response pairs if

> block-wise is involved.  DNS responses are provided in the body (which is the

Reviewer: ```suggestion
block-wise transfer is involved.  DNS responses are provided in the body (which is the
```
?

Reviewer: ```suggestion
block-wise is involved.  DNS responses are provided in the body (i.e. the
```
?

> payload, or the concatenated payloads) of CoAP response. A DoC server MUST

Reviewer: ```suggestion
payload, or the concatenated payloads) of a CoAP response. A DoC server MUST
```

or

```suggestion
payload, or the concatenated payloads) of the CoAP response. A DoC server MUST
```

Reviewer: This one is still there.

Author: Sorry, committed.
